src/app/pages/Collectibles/CollectiblePage/index.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { FC, useCallback, useMemo } from 'react';
+
+import { isDefined } from '@rnw-community/shared';
+import { useDispatch } from 'react-redux';
+
+import { FormSubmitButton, FormSecondaryButton, Spinner, Money, Alert } from 'app/atoms';
+import { useTabSlug } from 'app/atoms/useTabSlug';
+import PageLayout from 'app/layouts/PageLayout';
+import { loadCollectiblesDetailsActions } from 'app/store/collectibles/actions';
+import {
+  useAllCollectiblesDetailsLoadingSelector,
+  useCollectibleDetailsSelector
+} from 'app/store/collectibles/selectors';
+import { useTokenMetadataSelector } from 'app/store/tokens-metadata/selectors';
+import AddressChip from 'app/templates/AddressChip';
+import OperationStatus from 'app/templates/OperationStatus';
+import { TabsBar } from 'app/templates/TabBar';
+import { objktCurrencies } from 'lib/apis/objkt';
+import { BLOCK_DURATION } from 'lib/fixed-times';
+import { t, T } from 'lib/i18n';
+import { getAssetName } from 'lib/metadata';
+import { useAccount } from 'lib/temple/front';
+import { formatTcInfraImgUri } from 'lib/temple/front/image-uri';
+import { atomsToTokens } from 'lib/temple/helpers';
+import { TempleAccountType } from 'lib/temple/types';
+import { useInterval } from 'lib/ui/hooks';
+import { Image } from 'lib/ui/Image';
+import { navigate } from 'lib/woozie';
+
+import { useCollectibleSelling } from '../hooks/use-collectible-selling.hook';
+import { CollectiblesSelectors } from '../selectors';
+import { CollectiblePageImage } from './CollectiblePageImage';
+import { AttributesItems, PropertiesItems } from './TabsGridContent';
+
+const DETAILS_SYNC_INTERVAL = 4 * BLOCK_DURATION;
+
+interface Props {
+  assetSlug: string;
+}
+
+const CollectiblePage: FC<Props> = ({ assetSlug }) => {
+  const metadata = useTokenMetadataSelector(assetSlug);
+  const account = useAccount();
+
+  const { publicKeyHash } = account;
+  const accountCanSign = account.type !== TempleAccountType.WatchOnly;
+
+  const details = useCollectibleDetailsSelector(assetSlug);
+  const areAnyCollectiblesDetailsLoading = useAllCollectiblesDetailsLoadingSelector();
```

Comment:
Prettify yourself, you can group similar codes or close in meaning in small groups.
ex. All `Selectors` unite in one mini-block
```
...code

const metadata = useTokenMetadataSelector(assetSlug);
const details = useCollectibleDetailsSelector(assetSlug);
const areAnyCollectiblesDetailsLoading = useAllCollectiblesDetailsLoadingSelector();
 
...code
```

Reply:
it is not the rule but it makes your code more readable and beautifully for my opinion